DesignDoc/RMA.md

```diff
@@ -1,3 +1,205 @@
 # Request Merchandise Authorization (RMA) notification structure design
 
-## TBA
+## Intent
+
+Structure for RMA process.
+
+## Definitions
+
+- Consumer : Customers of device vendors (Eg: Google, Microsoft, X).
+- Publisher: Device vendor (Ciena, Adva, Infinera, X).
+
+## Features 
+
+RMA case is raised by the consumer to request the replacement of the defective parts. It consists of the following subflows:
+1. Consumer starts to request the RMA from the Publisher, and wait for the review/approval from the publisher
+1. Publisher ships the replacement, and notify the consumer about the shipment status of the replacement
+1. Consumer request techincial dispatch/schedule if needed
+1. Consumer ships the defective part for failure analysis, and notify the consumer about the shipment status of the defective parts
```

Comment:
Insert a points above:

1. Consumer triggers internal processes to receive shipped RMA part.
1. Consumer completes the RMA work with internal processes.